### ISSUE TITLE
关于teacher model 

### ISSUE THREAD

**Issue by Eutopia01** [jarucezh/cldfd#2]
感谢您的精彩工作！想要问一下在上传的文件中，sl_1gpu文件夹下的teacher model 是如何训练出来的，以及您所有的评估结果都是冻结主干，只训练线性层的结果吗？

**Comment by jarucezh** [comment]
你好！Teacher模型是在MiniImageNet模型上用CE loss训练出来的。测试阶段，所有的评估结果都只训练线性层。

***
(Let me translate it into English so that everyone can understand this Q&A.)

Q: How do you pre-train the teacher network? Did you only fine-tune the linear classifier with the frozen backbone in each evaluation result?

A: Hi, Eutopia01. The teacher model is pre-trained on MiniImageNet with CE Loss only. And, yes, the evaluation is conducted by fine-tuning the linear classifier layer only.

**Comment by Eutopia01** [comment]
好的，谢谢您！还想问一下，Teacher模型的训练用到了多少的计算资源以及多长时间呢？以及您提出的FEATURE DENOISING部分在训练线性层的时候对结果有明显提升，那如果在测试阶段不冻结主干，进行整体的微调，结果会有什么相应的变化吗？

**Comment by jarucezh** [comment]
你好！当时我们是用了一张P100，训练15个小时左右（没记错的话）。你提的是个好问题，也许后续工作中我们可以试一试！

***
Q: How many GPUs did you use to train the Teacher? Did you try to fine-tune using Feature Denoising with a trainable backbone during the evaluation stage?
A: We used 1 P100 to train the network for 15 hours. We haven't try this idea and that is a good question.

**Comment by Eutopia01** [comment]
好的，感谢您的回复！最后可以向您要一下测试阶段的代码吗？



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年11月14日 13:26 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [jarucezh/cldfd] 关于teacher model  (Issue #2) |

你好！当时我们是用了一张P100，训练15个小时左右（没记错的话）。你提的是个好问题，也许后续工作中我们可以试一试！

Q: How many GPUs did you use to train the Teacher? Did you try to fine-tune using Feature Denoising with a trainable backbone during the evaluation stage?
A: We used 1 P100 to train the network for 15 hours. We haven't try this idea and that is a good question.

**Comment by jarucezh** [comment]
我们当时用STARTUP的测试代码进行测试。很抱歉我们实验室服务器这两天坏了，等修好了以后我把Feature Denoising 的代码发给你。

***
Q: Could you provide the code of the evaluation, please?
A: We use the code of [STARTUP](https://github.com/cpphoo/STARTUP) to test. I am sorry that our lab server is down today. When it is recovered, let me send the function of Feature Denoising.

**Comment by Eutopia01** [comment]
好的好的，十分感谢！



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年11月14日 14:03 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [jarucezh/cldfd] 关于teacher model  (Issue #2) |

我们当时用STARTUP的测试代码进行测试。很抱歉我们实验室服务器这两天坏了，等修好了以后我把Feature Denoising 的代码发给你。
